airflow/executors/base_executor.py

```diff
@@ -540,7 +540,9 @@ def terminate(self):
         """Get called when the daemon receives a SIGTERM."""
         raise NotImplementedError
 
-    def cleanup_stuck_queued_tasks(self, tis: list[TaskInstance]) -> list[str]:  # pragma: no cover
+    def cleanup_stuck_queued_tasks(
+        self, tis: list[TaskInstance]
+    ) -> Iterable[TaskInstance]:  # pragma: no cover
```

Comment:
I'm a bit concerned about this return type change (str -> TI), and also the behavior change for this method. BaseExecutor is public, so we have to consider executors beyond those in our repos. And today, we expect other executors to fail tasks in this method.

Reply:
outdated